### ISSUE TITLE
[question] Bibliography?

### ISSUE THREAD

**Issue by kerim** [dcartertod/zotero-plugins#19]
The preview shows two sections, one called "bibliography" and the other called "individual citations." The first just says "undefined" but the "choose preview style" button only seems to change the "individual citations" section, not the bibliography section? Am I missing something?

**Comment by dcartertod** [comment]
Not necessarily. What is your current style (APA, MLA, etc.) overall for your Zotero?

**Comment by kerim** [comment]
Chicago

**Comment by dcartertod** [comment]
Just released an update 0.1.3 - should be available via Add-ons -> Check for updates

**Comment by kerim** [comment]
That fixes this. Thanks. (But there is a new problem, so I will open a new issue.)
